From 51fdcf4dccbd6975ff593e8d472562db8501c0bc Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Wed, 14 Oct 2015 19:18:15 +0800
Subject: [PATCH 0593/5242] MLK-11711-02 ARM: dts:
 imx7d-12x12-lpddr3-arm2-mqs: enable mqs sound
 card support in lpsr mode

commit  d99720c5ff67fbe2d5e5da6d3b7b85cc8b7506fa from
https://source.codeaurora.org/external/imx/linux-imx.git

enable mqs sound card support in lpsr mode

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-mqs.dts |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-mqs.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-mqs.dts
index 3a166a8..7fcdf6a 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-mqs.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-mqs.dts
@@ -24,8 +24,9 @@
 };
 
 &mqs {
-	pinctrl-names = "default";
+	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&pinctrl_mqs>;
+	pinctrl-1 = <&pinctrl_mqs>;
 	clocks = <&clks IMX7D_SAI1_ROOT_CLK>;
 	clock-names = "mclk";
 	status = "okay";
-- 
1.7.9.5

